radixdlt/src/main/java/com/radixdlt/consensus/Vertex.java

```diff
@@ -87,7 +111,7 @@ private Long getSerializerRound() {
 
 	@JsonProperty("round")
 	private void setSerializerRound(Long number) {
-		this.round = number == null ? null : Round.of(number.longValue());
+		this.round = number == null ? null : Round.of(number);
```

Comment:
Why remove the explicit unboxing here?

Reply:
IntelliJ suggestion as it is apparently unnecessary